### PR TITLE
Add attribution for NumFOCUS to README.md

### PR DESCRIPTION
**Pull request by rcurtin**
While waiting for my space station in Kerbal Space Program to achieve the right orbit, I modified the `README.md` to include attribution for NumFOCUS as described here: https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ functions as a "swiss army knife" for machine learning researchers.  In addition
...
+mlpack uses as [open governance model](./GOVERNANCE.md) and is fiscally
+sponsored by [NumFOCUS](https://numfocus.org/).  Consider making a
+[tax-deductible donation](https://numfocus.org/donate-to-mlpack) to help the
+project pay for developer time, professional services, travel, workshops, and a
+variety of other needs.
+
+<div align="center">
+  <a href="https://numfocus.org/donate-to-mlpack">
+    <img height="60px"
+         src="https://raw.githubusercontent.com/numfocus/templates/master/numfocus-logo.png"
```
Comment (zoq) [inline README.md]: https://github.com/numfocus/templates/pull/1

Comment (rcurtin) [reply README.md]: Thanks!  Fixed in 9f30003. :)

File: README.md [hunk 2]
```diff
@@ -34,6 +34,21 @@ functions as a "swiss army knife" for machine learning researchers.  In addition
 to its powerful C++ interface, mlpack also provides command-line programs and
 Python bindings.
 
+mlpack uses as [open governance model](./GOVERNANCE.md) and is fiscally
```
Comment (zoq) [inline README.md]: When reading this a second time the `as` sounds strange, the template uses `an` so I'm wondering if this is correct as well?

Comment (rcurtin) [reply README.md]: You're right, not sure how I missed that.  Fixed in 058915d.

**Pull request merged**